app/controllers/admins/calculators_controller.rb

```diff
@@ -6,7 +6,9 @@ class CalculatorsController < BaseController
 
     def index
       @calculators = if params[:search]
-                       Calculator.where('name LIKE ?', "%#{params[:search]}%")
+                       Calculator.all.where("name ILIKE ? OR slug ILIKE ?",
```

Comment:
`if params[:search]` if here is empty string like `""` then it returns nothing, but it's wrong behavior

Reply:
Fixed this, added `.blank?` to check if parameter is blank

Reply:
`!params[:search].blank?` => `params[:search].present?`

Reply:
1. you don't need `.all` here
2. I don't see tests for this behavior

Reply:
3. Remove this `if .. else` condition from controller, and use `scope` in model. Then use one `scope` here